docker-compose.yaml

```diff
@@ -373,9 +509,11 @@ volumes:
   kibana:
   operate_tmp:
   tasklist_tmp:
+  postgres-web:
 
 networks:
-  # Note there are two bridge networks: One for Camunda Platform and one for Identity.
-  # Identity and Keycloak are part of both as they need to be accessible by platform components.
+  # Note there are three bridge networks: One for Camunda Platform, one for Identity and one for Web Modeler.
+  # Identity and Keycloak are part of the first two as they need to be accessible by platform components.
```

Comment:
Let's also explain why certain Web modeler components are part of other networks here.